### PR TITLE
Implement Revocation API

### PR DESCRIPTION
**Pull request by hwrdtm**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by hwrdtm** [comment]
We'd talked about writing more notes or helper methods for the proof request and proof presentation steps as they are quite complicated currently, which I fully agree, but think that should be part of a separate PR. @wip-abramson

**Comment by wip-abramson** [comment]
Yep definitely I think there are a few helper methods to be added. Exactly how these should work is an open question, but something for a separate issue for sure.

File: tutorials/aries-basic-controller/docker-compose.yml [hunk 1]
```diff
@@ -16,24 +16,26 @@ services:
       - WALLET_TYPE=${WALLET_TYPE}
       - WALLET_KEY=${BOB_WALLET_KEY}
       - ACAPY_ADMIN_INSECURE_MODE=${BOB_ADMIN_SECURE}
+      - ACAPY_TAILS_SERVER_BASE_URL=tails-server
```
Comment (hwrdtm) [inline tutorials/aries-basic-controller/docker-compose.yml:19]: even though all containers are in the same `indy_demo` network, this `tails-server` doesn't resolve to the actual address of the `tails-server` container. Any ideas @wip-abramson ?

**Comment by wip-abramson** [comment]
Hey @hwrdtm, is this done? Does it need reviewing? Can't remember where we got to

File: tutorials/aries-basic-controller/docker-compose.yml (hunk 1 above)
Comment (wip-abramson) [reply tutorials/aries-basic-controller/docker-compose.yml]: Fixed this, needed http:tails-server:6543

**Comment by wip-abramson** [comment]
This is complete, awesome job @hwrdtm notebooks look great

**Pull request merged**